lib/compat/marshal.go

```diff
@@ -69,6 +69,8 @@ func UnmarshalInt(data []byte) (int, error) {
 	}
 
 	var num int32
+	// ignored to match ruby behavior
```

Comment:
Should confirm this is the case.

Reply:
The background on this ruby compatibility layer is that AuthN Server was originally built in Ruby and I wanted the Go version to be an easy upgrade. However at this point I think it's fair to say that the backwards compatibility challenge is with previous AuthN 1.x servers. Moving past this would probably require supporting two formats at once and then removing the old one in a 2.x release.

Reply:
I can try to update comment before merging.  Is there a known case where this would error and we want to proceed with zero?

Reply:
I'm not aware of any, no.

I just tried to search for how this gets used and came up short. Perhaps `UnmarshalInt` is irrelevant?

Reply:
I just fixed the comment for now maybe will take a pass over that file and remove anything that seems unused.  This is pretty far in the dark recesses, I certainly never saw it before and should familiarize 😄